CK2Plus_expanded/common/religion_features/00_features.txt

```diff
@@ -1549,9 +1549,15 @@ religion_doctrine = {
 			custom_tooltip = { text = tooltip_religion_feature_bloodthirsty_mechanics }
 			custom_tooltip = { text = tooltip_religion_feature_hard_conversion }
 			custom_tooltip = { text = tooltip_religion_feature_syncretic }
-			hidden_tooltip = { 
-				set_hard_to_convert = yes
+			hidden_tooltip = {
+				#set_hard_to_convert = yes
 				set_short_reign_opinion_year_mult = 0
+				law_set_allow_matrilineal_marriage = yes
+				set_feminist = yes
+				set_women_can_take_consorts = yes
+				set_men_can_take_consorts = no
+				set_female_temple_holders = yes
+				set_male_temple_holders = no

```

Comment:
Most of these are in the Basque religion already.

Reply:
I was just putting what I was told to put there instead lol do you have a suggestion for what to replace it with or should I just remove them entirely?

Reply:
I think the major points from `matriarchal clans` that Basque is still missing are rather the succession (enatic) changes, which are handled elsewhere mostly. I think it just needs the tooltip to show that change to the players. Another possibility is limiting the councillors as Whizzer said above, which again is done elsewhere and only needs a tooltip here.

We could also rethink whether by default Basque should be _this_ much feminist - I haven't done a lot of research, but it seems rather extreme to not just have women be equal, but actually superior in nigh every way. Did anyone do some extended research on them? Discord is down so I can't ask there.
I mean, an alternative would be to make them just "women friendly" (allowing women in any position of power/to have concubines) while unreformed, and for the reformation, make them embrace that distinction (to e.g. distinguish from the Christian anti-feminism) by going full female supremacy (i.e. banning men from positions of power/having concubines).

Reply:
And I also just randomly noted that both Celtic and Basque might need to have a `max_concubines` defined, as I'm not sure whether the game will default to the usual 3 or "unlimited" otherwise.
Again, Discord is down so I can't say that there.

Reply:
@LordPeter So far as "how feminist" they are, this is the religion, not the culture. Most things I've read about the Basques reinforces them being extremely matrilineal. I don't think making them enatic is too much of a stretch.

That being said, I'm not looking for feedback on what else needs done to the religions, overall. I'm just trying to get this particular pull request merged. I was intending on only doing a couple quick fixes that someone brought up and it kinda feels like you guys are trying to get me to fix everything that's not finished or needs tweaked in both religions. As such, can we please limit the feedback to what, specifically, needs changed in _this_ pull request, and in what way, to get it merged?

Reply:
Fair enough.
But you added these lines and as ?! pointed out they don't make sense in the current context. So if you want to quickly merge the pull request, you can either leave them in or take them out. Either way, we will need to look at Basque again. But we can do it in another branch.